luigi/execution_summary.py

```diff
@@ -270,15 +277,15 @@ def _get_comments(group_tasks):
     ("already_done", 'present dependencies were encountered'),
     ("completed", 'ran successfully'),
     ("failed", 'failed'),
-    ("scheduling_error", 'failed running complete() or requires()'),
+    ("scheduling_error", 'failed scheduling'),
     ("still_pending", 'were left pending, among these'),
     ("still_pending_ext", 'were missing external dependencies'),
     ("run_by_other_worker", 'were being run by another worker'),
     ("upstream_failure", 'had failed dependencies'),
     ("upstream_missing_dependency", 'had missing external dependencies'),
     ("upstream_run_by_other_worker", 'had dependencies that were being run by other worker'),
-    ("upstream_scheduling_error", 'had dependencies whose complete() or requires() failed'),
-    ("unknown_reason", 'were left pending because of unknown reason'),
+    ("upstream_scheduling_error", 'had dependencies whose scheduling failed'),
+    ("unknown_reason", 'did not run successfully because of unknown reason'),
```

Comment:
Alternatively, a valid wording would be "did not run", because that's equivalent to the common sense "left pending" (didn't enter `run()`). "Did not run successfully" is what I object against, because that can also be interpreted to mean that it did enter `run()`.